.travis.yml

```diff
@@ -0,0 +1,38 @@
+branches:
+  only:
+  - master
+  - /^v.*$/
+language: python
+matrix:
+  include:
+  - os: osx
+    language: generic
+    env: TERRYFY_PYTHON='homebrew 2'
+  - os: osx
+    language: generic
+    env: TERRYFY_PYTHON='homebrew 3'
+before_install:
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then git clone https://github.com/MacPython/terryfy;
+  fi
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source terryfy/travis_tools.sh; fi
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then get_python_environment $TERRYFY_PYTHON
+  venv; fi
+- if [[ "$TERRYFY_PYTHON" == "homebrew 3" ]]; then alias pip=`which pip3` ; fi
+install:
+- pip install numpy scipy
+- pip install -e .
+script:
+- nosetests
+after_success:
+- coveralls
+- echo [distutils]                                  > ~/.pypirc
+- echo index-servers = pypi                        >> ~/.pypirc
+- echo [pypi]                                      >> ~/.pypirc
+- echo repository=https://pypi.python.org/pypi     >> ~/.pypirc
+- echo username=muellner                           >> ~/.pypirc
+- echo password=$PYPIPASSWORD                      >> ~/.pypirc
+- if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then python setup.py bdist_wheel upload; fi
+env:
+  global:
+  - secure: ODOm9ZtMYY2+SmTY/cognMTC06hixMzRR7CWwHcCZnNJcmfC2+fEKhBLf/vigHGvnrd1fNsDN/QNX43yfdqkD6q7KAYMxn2BsUdbNvwcfwS2gk6J4nO8yzeas0aisar8WqU6sg3EZUw6WlrXIYsnnXVfTpAksGZ4kPeZNIM1Fq0=
```

Comment:
This is the encrypted environmental variable PYPIPASSWORD=your_pypi_password
Read here for more details https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables

You'll need to do this yourself like

`travis encrypt PYPIPASSWORD=your_pypi_password`

Using the travis command line tool.